### DISCUSSION TITLE
Syntax Error on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by Voltbolzen** [swar/Swar-Chia-Plot-Manager#440] [Q&A]
Hi,

i tried to use the plot manager on Ubuntu 20.04.
And i get the following Error:

> :~/Swar-Chia-Plot-Manager-main# python manager.py start
Traceback (most recent call last):
  File "manager.py", line 4, in <module>
    from plotmanager.library.utilities.commands import start_manager, stop_manager, view, analyze_logs
  File "/root/Swar-Chia-Plot-Manager-main/plotmanager/library/utilities/commands.py", line 15
    from plotmanager.library.utilities.print import print_view
                                           ^
SyntaxError: invalid syntax

> 
i dont know whats the problem. On my Windows system everything works well.

**Comment by swar** [comment]
Wrong version of python.
